webui/react/src/types.ts

```diff
@@ -337,6 +337,7 @@ export interface ExperimentItem {
   name: string;
   numTrials: number;
   progress?: number;
+  resourcePool?: string
```

Comment:
are these shared with the other pr? #1831 same comment about whether we need to have these as optional attributes

Reply:
Same response to the other comment. This will allow us to point the webui to an older version of the master. It doesn't need to be optional if it is pointing to a new version of the master. I can change it to required if we want.

Reply:
we discussed it in the sync we're going to have it be required (overall after the two prs land) as webui won't point to older version of master.